templates/taxonomy-module.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * The template for displaying a module.
+ *
+ * Override this template by copying it to your_theme/sensei/taxonomy-module.php.
```

Comment:
Is this going to be a backwards incompatibility for anyone who was previously overriding the `archive-lesson.php` template? Do we know if anyone even does that?

Reply:
Yeah, that's a really good point! I've made a note to mention this in the release post. If someone wants to revert back to the old layout, they'll be able to copy the content of `archive-lesson.php` and paste it into `your_theme/sensei/taxonomy-module.php`.